modoboa/admin/management/commands/subcommands/_manage_dkim_keys.py

```diff
@@ -33,7 +33,7 @@ def create_new_dkim_key(self, domain):
         if code:
             print("Failed to generate DKIM public key for domain {}: {}"
                   .format(domain.name, output))
-        public_key = ""
+        public_key = b""
         for cpt, line in enumerate(output.splitlines()):
```

Comment:
It would be easier to convert output to text
`for cpt, line in enumerate(smart_text(output).splitlines()):`